packages/terra-worklist-data-grid/src/WorklistDataGrid.jsx

```diff
@@ -210,11 +216,11 @@ function WorklistDataGrid(props) {
   // -------------------------------------
   // functions
 
-  const isRowSelectionCell = (columnIndex) => (
+  const isRowSelectionCell = useCallback((columnIndex) => (
```

Comment:
We should not need to use useCallback on this function.  There is not a need to memoize the function.

Reply:
Removed